### PR TITLE
Update the doc with important consideratinos

### PR DESCRIPTION
**Pull request by letzya**

### CODE REVIEW

**Comment by letzya** [comment]
@marksou can you please merge?

**Comment by marksou** [comment]
@letzya Can you approve please?

File: tyk-docs/content/shared/api-def-custom-analytics.md [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
-Since we are basically declaring headers' prefix as tags Tyk as expected from tags will also do aggregations per tag for the API calls.
-If you don't want to have aggregation for these tags you can add them or their prefixes to `ignore_tag_prefix_list` in `pump.conf` in case the pump is writing the aggregated analytics to MongoDB. Alternatively, in case MDCB is doing the writing to mongoDB, set the same field in `tyk_sink.conf` at root level (please note that this field is replacing `"aggregates_ignore_tags"` which is still working but will eventually be deprecated).
+This functionality is useful if you need to record additional information from the request in the analytics, without enabling detailed logging, which records the full request and response objects and as such consumes a lot of space.
 
-See [Log Browser](/docs/analytics-and-reporting/log-browser/) for more details on setting via the Dashboard.
+**Important considerations**
+- Since we are basically declaring headers' prefix as tags, Tyk, as expected from tags, will `also` add an aggregation point per tag in the aggregated analytics.
```
Comment (craigtyk) [inline tyk-docs/content/shared/api-def-custom-analytics.md]: I'd remove "basically" and "as expected from tags" as unnecessary

**Review by craigtyk — CHANGES_REQUESTED** [review]
small change suggested on line 11

File: tyk-docs/content/shared/api-def-custom-analytics.md (hunk 1 above)
Comment (marksou) [reply tyk-docs/content/shared/api-def-custom-analytics.md]: Have removed

**Review by craigtyk — APPROVED** [review]
lgtm, thanks

**Pull request merged**